lib/season_statistics.rb

```diff
@@ -1,15 +1,108 @@
 module SeasonStatistics
 
 
-  def biggest_bust(season)
-    games_by_team = @game_teams.group_by do |game_team|
-      game_team.team_id
+  def biggest_bust(season_id)
+    filtered_games = []
+    @games.each do |game_id, game|
+      if game.season == season_id
+        filtered_games.push(game)
+      end
```

Comment:
Once we've created a helper method for `filtered_games`, we'll be able to shrink this.